server/README.md

```diff
@@ -0,0 +1,22 @@
+# 🥏 UltimateApp Server
+
+Server to host the drills for ultimate app based on Mongodb, ApolloServer and Graphql.
+
+## ⌨️ Development
+
+You need a recent version of nodejs, and a [running local mongodb server](https://docs.mongodb.com/manual/installation/#mongodb-community-edition-installation-tutorials)
+
+```
+git clone git@github.com:UltimateApp/UltimateApp.git
```

Comment:
I'm wondering whether it should be stored in another repository? What are the good practices on this?

My first inclination would be to have 1 repo for 1 app

What do you think about it ?

Reply:
> I'm wondering whether it should be stored in another repository? What are the good practices on this?
> 
> My first inclination would be to have 1 repo for 1 app
> 
> What do you think about it ?


My opinion on that is as long as it does not become too complex, and everyone works on both parts, there are some good sides to keep it in one repo:
- everybody have an easy access to both components --> better for a good understanding of the whole thing
- no need to make 2 different PRs to introduce some functional changes
- no need of synchronisation for build/test/deployment

Moreover we can still split it later if we feel having one repo is too heavy

Reply:
wow that's a lot of good arguments in favor of keeping it here. Makes sense :)